src/internal.ts

```diff
@@ -109,7 +109,7 @@ function initIntercoms() {
 
       const portFrame = incomingPort.sender.frameId
 
-      if (portFrame)
+      if (portFrame !== undefined)
```

Comment:
This would allow nulls to pass this check. Should this be `portFrame || portFrame === 0` instead?